### ISSUE TITLE
"NameError: uninitialized constant" installing 0.90 with homebrew (MacOs Mohave)

### ISSUE THREAD

**Issue by dfrese** [philpennock/sieve-connect#11]
Hello,

I'm getting the following error trying to install via `brew install sieve-connect` on MacOS 10.14.4

```
==> Installing sieve-connect from philpennock/protocols
==> Downloading https://people.spodhuis.org/phil.pennock/software/sieve-connect-0.90.tar.bz2
Already downloaded: ....
Error: An exception occurred within a child process:
  NameError: uninitialized constant #<Class:0x00007fc5021ac6a0>::DATA
Did you mean?  Data
               Date
```

`brew install --verbose` does not show something valuable, methinks (only untar,cp,chmod lines)

**Comment by philpennock** [comment]
I can reproduce, with uninstall and install.  This appears to be the patches and `DATA` reference to post-`__END__` content.  I suspect a backwards incompatible change in Homebrew (since this worked a week ago!), I'll investigate now.

Thanks for the report!

**Comment by philpennock** [comment]
Fixed in philpennock/homebrew-protocols@5cf2d1e73451aad7fc1f9cdcb8d0cc0550cee33f

**Comment by dfrese** [comment]
Thanks!
